products/test/conftest.py

```diff
@@ -0,0 +1,70 @@
+import pytest
+import redis
+
+from products.dependencies import REDIS_URI_KEY
+
+
+@pytest.fixture
+def redis_config():
+    return {REDIS_URI_KEY: 'redis://localhost:6379/11'}
```

Comment:
Could we grab this out of `config.yaml` so that users can override it when running tests locally?